source/features/latest-tag-button.tsx

```diff
@@ -37,26 +39,59 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 	return tags[0];
 }, {
 	expiration: 1,
-	cacheKey: () => __featureName__ + ':' + getRepoURL()
+	cacheKey: () => __featureName__ + '_tags:' + getRepoURL()
 });
 
-function getTagLink(latestRelease: string): HTMLAnchorElement {
+async function currentBranchIsBleedingEdge(latestTag: string, defaultBranch: string, currentBranch: string): Promise<boolean> {
+	// Only default branch can be cutting edge (i.e., ahead of latest release)
+	if (currentBranch !== defaultBranch) {
+		return false;
+	}
+
+	// Find the last commits from the default branch and latest tag
+	const repository = await api.v4(`
+		repository(${getRepoGQL()}) {
+			latestTag:object(expression: "${latestTag}") {
+				... on Commit {
+					oid
+				}
+			  },
+			  defaultBranch:object(expression: "${defaultBranch}") {
+				... on Commit {
+					oid
+				}
+			}
+		}
+	`);
+	// If the default branch has a different (later) commit, it's considered bleeding edge
+	return repository.repository.defaultBranch!.oid !== repository.repository.latestTag!.oid;
+}
+
+async function getTagLink(latestRelease: string): Promise<HTMLAnchorElement> {
 	const link = <a className="btn btn-sm btn-outline tooltipped tooltipped-ne ml-2">{tagIcon()}</a> as unknown as HTMLAnchorElement;
 
 	const currentBranch = getCurrentBranch();
-
+	const defaultBranch = await getDefaultBranch();
+	const [isBleedingEdge] = await Promise.all([
+		currentBranchIsBleedingEdge(latestRelease, defaultBranch, currentBranch)
```

Comment:
Let’s drop **every** reference to “bleeding edge” and just use “ahead” or “behind”

“$branch is ahead of the latest release”

Reply:
Done